docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/crowdstrike-filevantage.md

```diff
@@ -124,3 +124,4 @@ Sources can be configured using UTF-8 encoded JSON files with the Collector Ma
 ### Limitation
 
 - This source supports a maximum of 65000 resourceIDs. Exceeding this resourceIDs limit may cause the source to return a `FIRST-PARTY-GENERIC` error type.
+- The polling interval value should be atleast one minute.
```

Comment:
```suggestion
```

Reply:
Add this in the Configuration Object section description.

Reply:
Done.